--- kpackage-0.0.1/src/kpackage/packageloader.cpp.omv~	2014-12-10 01:00:28.728672010 +0100
+++ kpackage-0.0.1/src/kpackage/packageloader.cpp	2014-12-10 01:00:41.967506501 +0100
@@ -230,7 +230,7 @@ QList<KPluginMetaData> PackageLoader::li
             for (QJsonArray::const_iterator iter = plugins.constBegin(); iter != plugins.constEnd(); ++iter) {
                 const QJsonObject &obj = QJsonValue(*iter).toObject();
                 const QString &pluginFileName = obj.value(QStringLiteral("FileName")).toString();
-                const KPluginMetaData m(obj, QString(), pluginFileName);
+                const KPluginMetaData m(obj, pluginFileName);
                 if (m.isValid()) {
                     lst << m;
                 }
